packages/salesforcedx-vscode-core/src/commands/templates/apexGenerateUnitTestClass.ts

```diff
@@ -20,13 +21,31 @@ import { getParamGatherers } from './apexGenerateClass';
 import { LibraryApexGenerateUnitTestClassExecutor } from './executors/LibraryApexGenerateUnitTestClassExecutor';
 import { APEX_CLASS_TYPE } from './metadataTypeConstants';
 
+// if called from a file's context menu, will deliver the clicked file URI and an array of selected files
+// if called from the command pallet args will be empty
 export const apexGenerateUnitTestClass = async (
-  unitFileToCreate?: string,
-  unitFileDirectory?: string,
+  outputDirectory?: string | vscode.Uri,
```

Comment:
This is the risky bit. I changed where I found usage but it looks like it is only here for testing. If it is somewhere that I didn't find then we can back this out.

The reason for this is it better matches the arg ordering when invoked from the Context Menu. Ignoring the second parameter's variation but still adding it to typings so we know it is possible.